test/IntegrationTests/test_getallchats.py

```diff
@@ -0,0 +1,779 @@
+        self.assertIn("no chats found for this customer and chat id", response.text.lower(),
+                      "Expected error message not found in the response.")
+
+        logger.info("=== Test Case 10 Completed ===\n")
+
+    def test_large_page_size(self):
+        """Test case 11: Large Page Size"""
+        logger.info("=== Starting Test Case 11: Large Page Size ===")
+
+        local_chat_id = self.create_chat()  # Create a new chat and get the chat_id
+        # Add multiple chat messages to the chat
+        messages_to_add = [
+            "How do I contact support?",
+            "What are the payment options?",
+            "Can I change my order?",
+            "How do I track my order?",
+            "What is your privacy policy?",
+        ]
+
+        # This is the static response you are currently getting from the API
+        static_response = "You will get the correct answer once AI is integrated."
+
+        chat_url = f"{self.BASE_URL}/chat"
+        for message in messages_to_add:
+            message_data = {
+                "customer_guid": self.valid_customer_guid,
+                "chat_id": local_chat_id,  # Include the chat_id here
+                "question": message  # Ensure this key matches what the API expects
+            }
+            logger.info(f"INPUT: Creating message with data: {str(message_data)}")
+            message_response = requests.post(chat_url, json=message_data)
+            logger.info(f"OUTPUT: Message creation response status: {message_response.status_code}")
+            self.assertEqual(message_response.status_code, 200, "Failed to create test message.")
+
+        # Now retrieve all chats using the correct chat_id
+        url = f"{self.BASE_URL}/getallchats"
+        data = {
+            "customer_guid": self.valid_customer_guid,
+            "chat_id": local_chat_id,  # Use the chat_id from the initial chat creation
+            "page": 1,
+            "page_size": 1000
+        }
+        logger.info(f"INPUT: Sending request with data:\n{str(data)}")
+
+        response = requests.post(url, json=data)
+        logger.info(f"OUTPUT: Response status code: {response.status_code}")
+        logger.info(f"OUTPUT: Response content: {response.text}")
+
+        # Check for successful response
+        self.assertEqual(response.status_code, 200, "Failed to retrieve chats.")
+
+        response_data = response.json()
+
+        # Check that 'messages' is a list
+        messages = response_data.get('messages')
+        self.assertIsInstance(messages, list, "Expected 'messages' to be a list.")
+
+        # Verify that all added messages are in the response
+        for expected_message in messages_to_add:
+            # Check for user message
+            self.assertTrue(any(msg.get('message') == expected_message for msg in messages),
+                            f"Expected message '{expected_message}' not found in the response.")
+
+            # Check for corresponding system response
```

Comment:
We may different responses later for answers
So we can remove this check - we can remove the below three lines

```
# Check for corresponding system response
self.assertTrue(any(msg.get('message') == static_response for msg in messages),
                            f"Expected response for '{expected_message}' not found in the response.")

```

Reply:
Done, sir. I removed the three lines in this test case.